clients/admin-ui/src/features/privacy-requests/PrivacyRequestsContainer.tsx

```diff
@@ -16,17 +19,20 @@ const PrivacyRequestsContainer: React.FC = () => {
   const { processing } = useDSRErrorAlert();
   const [revealPII, setRevealPII] = useState(false);
 
+  const { plus: hasPlus } = useFeatures();
+
   useEffect(() => {
     processing();
   }, [processing]);
 
   return (
     <>
-      <Flex data-testid="privacy-requests">
+      <Flex data-testid="privacy-requests" gap={4}>
         <Heading mb={8} fontSize="2xl" fontWeight="semibold">
           Privacy Requests
         </Heading>
         <Spacer />
+        {hasPlus ? <SubmitPrivacyRequest /> : null}
```

Comment:
We need to also constrain this to only users that have the `PRIVACY_REQUEST_CREATE` scope